flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples/windowing/SessionWindowing.scala

```diff
@@ -60,11 +64,9 @@ object SessionWindowing {
         input.foreach(value => {
           ctx.collectWithTimestamp(value, value._2)
           ctx.emitWatermark(new Watermark(value._2 - 1))
-          if (!fileOutput) {
-            println(s"Collected: ${value}")
-          }
+          if (!fileOutput) println(s"Collected: ${value}")
```

Comment:
AFAIK we encourage adding braces even for one line if statements, so the previous version was better.